styles/components/alert.css

```diff
@@ -0,0 +1,35 @@
+.alert {
+  background: rgba(255, 255, 255, 0.3);
+  backdrop-filter: blur(6.5px);
+  -webkit-backdrop-filter: blur(6.5px);
+  padding: 15px 20px;
```

Comment:
either use rem or px dont go for combination (if the size is less than 1rem you can use px there)

Reply:
yes understood